src/features/annotation-thread/AnnotationThread.scss

```diff
@@ -0,0 +1,10 @@
+@import 'src/elements/common/variables';
```

Comment:
Since it's no longer under `elements`, we could use the `src/styles/variables` file.